widget/editor.go

```diff
@@ -69,15 +69,25 @@ type Editor struct {
 		// position when moving between lines.
 		xoff fixed.Int26_6
 
-		// line is the caret line position as an index into lines.
-		line int
-		// col is the caret column measured in runes.
-		col int
-		// (x, y) are the caret coordinates.
-		x fixed.Int26_6
-		y int
+		// pos.lineCol.Y is the caret line position as an index into lines.
+		// pos.lineCol.X is the caret column measured in runes.
+		// pos.(x, y) are the caret coordinates.
+		// pos.ofs is the caret offset into the text, from rr.caret
```

Comment:
These comments, while useful, seem redundant or misplaced. Why not have them on combinedPos?